src/measure_theory/function/lp_space.lean

```diff
@@ -790,6 +790,20 @@ begin
   exact snorm'_add_lt_top_of_le_one hf.1 hf.2 hg.2 hp_pos hp1_real,
 end
 
+lemma ae_le_of_snorm_ess_sup_le {f : α → F} {x : ℝ≥0∞} (hx : snorm_ess_sup f μ ≤ x)
+  (hf : is_bounded_under (≤) μ.ae (λ y, ‖f y‖₊) . is_bounded_default) : ∀ᵐ y ∂μ, ↑‖f y‖₊ ≤ x :=
```

Comment:
You shouldn't need `hf` here, as `snorm_ess_sup` is the ess_sup taking in ennreal, which is complete.